Comanche055/JET_SELECTION_LOGIC.agc

```diff
@@ -177,7 +177,7 @@ PYTABLE		OCT	0		# 0	0		0
 		OCT	2421		# +	+		3
 		OCT	2610		# -	+		3
 		OCT	0146		# 0	-		6
-		OCT	2504		# + 	-		6
+		OCT	2504		# + -		6
```

Comment:
This is not right: http://www.ibiblio.org/apollo/ScansForConversion/Comanche055/1042.jpg